audio/whisper/Portfile

```
@@ -24,36 +24,42 @@ output.wav` to convert the file first.
 "
 
 checksums           whisper.cpp-${version}.tar.gz \
-                    rmd160  26b0b29e46c990b13c9d10d95c11aa429fc8d56d \
-                    sha256  da7988072022acc3cfa61b370b3c51baad017f1900c3dc4e68cb276499f66894 \
-                    size    5088298
+                    rmd160  211e55a336132a878471382d8002425f069db683 \
+                    sha256 2fda42b57b7b8427d724551bd041616d85401fb9382e42b0349132a28920a34f \
+                    size    6216643 
 
-build.target        main
-makefile.has_destdir    no
 patchfiles          whisper-add-models.diff \
+                    whisper-add-depends.diff \
                     ggml-hard-coded.diff \
                     add-metal-path.diff
 
 post-patch {
     reinplace "s|__PREFIX__|${prefix}|" \
-        whisper.cpp \
-        ggml-metal.metal \
-        ggml-metal.m
+        src/whisper.cpp \
+        ggml/src/ggml-metal/ggml-metal.metal \
+        ggml/src/ggml-metal/ggml-metal.m
 }
 
 post-build {
-    move ${worksrcpath}/main ${worksrcpath}/whisper
+    move ${workpath}/build/bin/whisper-cli ${workpath}/build/bin/whisper
 }
 
 destroot {
     xinstall -d ${destroot}${prefix}/bin
-    xinstall -m 755 ${worksrcpath}/whisper ${destroot}${prefix}/bin/whisper
+    xinstall -m 755 ${workpath}/build/bin/whisper ${destroot}${prefix}/bin/whisper
```

You can probably eliminate that entire `post-build` section by doing something like:

```suggestion
    xinstall -m 0755 ${workpath}/build/bin/whisper-cli ${destroot}${prefix}/bin/whisper
```